_docs/README.md

```diff
@@ -164,7 +176,11 @@ variables or dependencies will be available.
 ```yaml
 variables:
   - name: <NAME>
-    prompt: <PROMPT>
+    description: <DESCRIPTION>
+    type: <TYPE>
+    options:
+      - <CHOICE>
+      - <CHOICE>
     default: <DEFAULT>
```

Comment:
Perhaps. I'm not convinced an extra parameter and juggling extra data offers a lot of benefit in this case though.